src/app/invite/[token]/page.tsx

```
@@ -0,0 +1,358 @@
+              <div className="space-y-2">
+                <div className="flex justify-between items-center">
+                  <span className="text-sm text-gray-600">Store:</span>
+                  <span className="font-medium">{invitation?.name}</span>
+                </div>
+                <div className="flex justify-between items-center">
+                  <span className="text-sm text-gray-600">Email:</span>
+                  <span className="font-medium">{invitation?.email}</span>
+                </div>
+                <div className="flex justify-between items-center">
+                  <span className="text-sm text-gray-600">Role:</span>
+                  <RoleBadge role={invitation!.role} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)